x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_setup/index.test.tsx

```diff
@@ -65,11 +65,11 @@ describe('useConnectorSetup', () => {
       ).toEqual([
```

Comment:
nit: `expect.objectContaining()` could be used here to avoid data modification via `map`.